GitUI/Editor/FileViewer.cs

```diff
@@ -982,22 +969,10 @@ private void encodingToolStripComboBox_SelectedIndexChanged(object sender, Event
             Encoding encod = null;
             if (string.IsNullOrEmpty(encodingToolStripComboBox.Text))
                 encod = Module.FilesEncoding;
-            else if (encodingToolStripComboBox.Text.StartsWith("Default", StringComparison.CurrentCultureIgnoreCase))
-                encod = Encoding.Default;
```

Comment:
Yes you right, but it should be saved to config file as "Default" because we have portable GitExt that should use the same config on several computers. If user wants to use specific encoding on different computer he can add it later manually to list.